src/codeu/chat/server/Server.java

```diff
@@ -172,6 +176,24 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
+    //Gets the Server information
+    this.commands.put(NetworkCode.SERVER_VERSION_REQUEST, new Command(){
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
+
```

Comment:
As @mlk209 pointed out, there are a bunch of functions with leading blank lines, never mind what https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace says.

So, never mind this review comment.